### PR TITLE
Send all locha-p2p RuntimeEvents to Java

### PR DESCRIPTION
**Pull request by jeandudey**
Almost finished, Java method signatures need to be updated in order for this PR to be ready for review.

### CODE REVIEW

File: android/app/src/main/java/io/locha/p2p/runtime/ChatServiceEvents.java [hunk 1]
```diff
@@ -17,18 +17,123 @@
...
+    public void onListenerError(String error);
+
+    /**
+     * Dialing a peer
+     *
+     * @param peer The peer that is being dialed.
      */
-    public void onNewListenAddr(String multiaddr);
+    public void onDialing(String peer);
 }
```
Comment (luisan00) [inline android/app/src/main/java/io/locha/p2p/runtime/ChatServiceEvents.java:139]:
```suggestion
}

```

File: android/app/src/main/java/io/locha/p2p/runtime/ChatServiceEvents.java [hunk 2]
```diff
@@ -17,18 +17,123 @@
...
+     * Error ocurred on incoming connection.
+     *
+     * @param localAddr Our local address where the connection was received.
+     * @param sendBackAddr The address we use to send back to this peer.
+     * @param error The error cause.
+     */
+    public void onIncomingConnectionError(String localAddr, String sendBackAddr, String error);
+
+    /**
+     * Peer banned
```
Comment (luisan00) [inline android/app/src/main/java/io/locha/p2p/runtime/ChatServiceEvents.java:81]:
```suggestion
     * A peer was banned
```

File: android/app/src/main/java/io/locha/p2p/runtime/ChatServiceEvents.java [hunk 3]
```diff
@@ -17,18 +17,123 @@
...
+     *
+     * @param peer The peer banned.
+     */
+    public void onBannedPeer(String peer);
+
+    /**
+     * Unreachable peer address
+     *
+     * @param peer The unreachable peer.
+     * @param address The unreachable address for peer.
```
Comment (luisan00) [inline android/app/src/main/java/io/locha/p2p/runtime/ChatServiceEvents.java:91]:
```suggestion
     * @param address The unreachable address for the given peer.
```

File: android/app/src/main/java/io/locha/p2p/runtime/ChatServiceEvents.java [hunk 4]
```diff
@@ -17,18 +17,123 @@
...
+
+    /**
+     * A listener has been closed
+     *
+     * @param addresses The local addresses closed.
+     */
+    public void onListenerClosed(String[] addresses);
+
+    /**
+     * An error happened whithing a listener
```
Comment (luisan00) [inline android/app/src/main/java/io/locha/p2p/runtime/ChatServiceEvents.java:127]:
```suggestion
     * An error happened within a listener
```

**Review by luisan00 — CHANGES_REQUESTED** [review]

**Review by xkevin190 — APPROVED** [review]

**Comment by jeandudey** [comment]
@luisan00 comments addressed. Thanks for the review :wink:

**Review by luisan00 — APPROVED** [review]

**Pull request merged**